test/Microsoft.Dnx.TestHost.Tests/TestHostTest.cs

```diff
@@ -22,10 +22,10 @@ public class TestHostTest
         public TestHostTest()
         {
             var libraryManager = PlatformServices.Default.LibraryManager;
-            _testProject = Path.GetDirectoryName(libraryManager.GetLibrary("Sample.Tests").Path);
+            _testProject = "Not set"; // _testProject = Path.GetDirectoryName(libraryManager.GetLibrary("Sample.Tests").Path);
         }
 
-        [ConditionalFact]
+        [Fact(Skip = "TestHost has to be redesigned to support the new project format")]
         [OSSkipCondition(OperatingSystems.Linux)]
         [OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
Sorry, I wasn't clear. Meant you should remove the `[OSSkipCondition]`s as well.